src/test/java/net/sf/jabref/exporter/layout/format/HTMLCharsTest.java

```diff
@@ -21,14 +21,14 @@ public void testBasicFormat() {
 
         Assert.assertEquals("h&aacute;llo", layout.format("h\\'allo"));
 
-        Assert.assertEquals("&#305; &#305;", layout.format("\\i \\i"));
-        Assert.assertEquals("&#305;", layout.format("\\i"));
-        Assert.assertEquals("&#305;", layout.format("\\{i}"));
-        Assert.assertEquals("&#305;&#305;", layout.format("\\i\\i"));
+        Assert.assertEquals("&imath; &imath;", layout.format("\\i \\i"));
```

Comment:
Historical reasons. Earlier the huge conversion list resided in that file, so adding a method "formatUnicode" was simple. Indeed there should (very soon) be a UnicodeToLatexConverter extracted... Doing this is steps...